preamble/dissemination.tex

```diff
@@ -1,4 +1,78 @@
 \chapter*{Dissemination}
 \addcontentsline{toc}{chapter}{Dissemination}
 
-This is a summary of how this research has been disseminated.
+\section*{Publications}
+
+\begin{itemize}
+    \item H. Wilde, V. Knight, and J. Gillard. \textbf{Evolutionary dataset
+        optimisation: learning algorithm quality through evolution}.
+        \emph{Applied Intelligence}, 50(4):1172–1191,
+        2020.~\doi{10.1007/s10489-019-01592-4}.
+    \item H. Wilde, V. Knight, and J. Gillard. \textbf{Matching: A Python
+        library for solving matching games}. \emph{Journal of Open Source
+    Software}, 5(48):2169, 2020.~\doi{10.21105/joss.02169}.
+\end{itemize}
+
+\section*{In preparation}
+
+\begin{itemize}
+    \item H. Wilde. \textbf{A review of current literature covering the
+        intersections of clustering, algorithm evaluation and healthcare
+        modelling}. Available on GitHub at
+        \github{daffidwilde/literature-review}.
+    \item H. Wilde, V. Knight, and J. Gillard. \textbf{A novel initialisation
+        based on hospital-resident assignment for the \(k\)-modes algorithm}.
+        Preprint available at \arxiv{2002.02701} --- \emph{submitted to
+        Knowledge and Information Systems}.
+    \item H. Wilde, V. Knight, J. Gillard, and K. Smith. \textbf{Segmentation
+        analysis and the recovery of queuing parameters via the Wasserstein
+        distance: a study of administrative data for patients with chronic
+        obstructive pulmonary disease}. Preprint available at \arxiv{2008.04295}
+        --- \emph{under review at the European Journal of Operational Research}.
```

Comment:
Move these 2 to a `\section*{Under review}` section

Reply:
Good idea. Moved in 953a0af.